README.md

```diff
@@ -1,131 +1,75 @@
-# BoldBI Embedding Angular with ASP.NET Core Samples
+# Bold BI Embedded Sample in Angular with ASP.NET Core
 
- This Bold BI Angular with Asp.Net Core sample contains Angular 14 as front-end sample and Asp.Net Core as back-end sample. This sample demonstrates the dashboard rendering with the list of dashboards available in your Bold BI server.
+This project was created using ASP.NET Core 6.0. The purpose of this application is to demonstrate how to render the dashboard available on your Bold BI server.
 
-This section guides you in using the Bold BI dashboard in your Angular with ASP.NET Core sample application.
+## Dashboard view
 
- * [Requirements to run the demo](#requirements-to-run-the-demo)
- * [Using the Angular with ASP.NET Core sample](#using-the-Angular-with-ASP.NET-Core-sample)
- * [Online Demos](#online-demos)
- * [Documentation](#documentation)
+![Dashboard View](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
- ## Requirements to run the demo
-
-The samples require the following requirements to run.
-
- * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
+ ## Requirements/Prerequisites
+ 
  * [.NET Core 6.0](https://dotnet.microsoft.com/en-us/download/dotnet-core)
  * [Node.js](https://nodejs.org/en/)
- * [Visual Studio Code](https://code.visualstudio.com/download)
 
- ## Using the Angular with ASP.NET Core sample
+ > **NOTE:** Node.js versions 14.16 to 18.16 are supported.
+
+ #### Help link
+
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+  * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/embedded-bi/site-administration/embed-settings/#get-embed-configuration-file) to enable it.
```

Comment:
Change the link on **instructions** as mentioned in previously reviewed PR's

Reply:
Instruction link changed